app.py

```diff
@@ -104,6 +105,40 @@ def load_adventure_assignments_per_level(lang, level):
 
 
 app = Flask(__name__, static_url_path='')
+
+# set the security stuff https://github.com/GoogleCloudPlatform/flask-talisman
+content_security_policy = {
+    'default-src': [
+        '\'self\'',
+        '*.cloudfront.net',
+        'youtube.com',
+    ],
+    'font-src': [
+        'fonts.googleapis.com',
+        'fonts.gstatic.com',
+    ],
+    'frame-src': [
+        'www.youtube.com',
+    ],
+    'img-src': [
+        '\'self\'',
+        'data:',
+    ],
+    'script-src': [
+        '\'self\'',
+        's3.amazonaws.com',
+        '\'unsafe-inline\'',
+        '\'unsafe-eval\'',
+    ],
+    'style-src': [
+        '\'self\'',
+        '\'unsafe-inline\'',
```

Comment:
this is another insecure practice, and the comments are similar to those above.

Reply:
Can you elaborate on the unsafety of inline styles? Keen to be educated.

Reply:
This is definitely one of the more exotic ones, and while most modern browsers won't execute any javascript in CSS attributes (like `style=background-image:url('javascript:alert("hi")')`), it _could_ be an issue for super old browsers, but I'm also going to recommend we keep it, for similar reasons as the `script-src` ones, to be expanded on below.